metapool/tests/data/runs/191103_D32611_0365_G00DHB5YXX/sample-sheet.csv

```diff
@@ -17,15 +17,15 @@ Chemistry,Default,,,,,,,,,
 ReverseComplement,0,,,,,,,,,
 ,,,,,,,,,,
 [Data],,,,,,,,,,
-Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,I5_Index_ID,index2,Sample_Project,Well_description
-1,sample1,sample1,FooBar_666_p1,A1,iTru7_107_07,CCGACTAT,iTru5_01_A,ACCGACAA,Baz,importantsample1
-1,sample2,sample2,FooBar_666_p1,A2,iTru7_107_08,CCGACTAT,iTru5_01_A,CTTCGCAA,Baz,importantsample2
-3,sample1,sample1,FooBar_666_p1,A3,iTru7_107_09,GCCTTGTT,iTru5_01_A,AACACCAC,Baz,importantsample1
-3,sample2,sample2,FooBar_666_p1,A4,iTru7_107_10,AACTTGCC,iTru5_01_A,CGTATCTC,Baz,importantsample2
-3,sample31,sample31,FooBar_666_p1,A5,iTru7_107_11,CAATGTGG,iTru5_01_A,GGTACGAA,FooBar_666,importantsample31
-3,sample32,sample32,FooBar_666_p1,B6,iTru7_107_12,AAGGCTGA,iTru5_01_A,CGATCGAT,FooBar_666,importantsample32
-3,sample34,sample34,FooBar_666_p1,B8,iTru7_107_13,TTACCGAG,iTru5_01_A,AAGACACC,FooBar_666,importantsample34
-3,sample44,sample44,Baz_p3,B99,iTru7_107_14,GTCCTAAG,iTru5_01_A,CATCTGCT,Baz,importantsample44
+Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,I5_Index_ID,index2,Sample_Project,well_description
```

Comment:
Changes to values in the Sample_ID, Sample_Name, and well_description columns was to confirm that strings derived from these values were getting them from the right column. At least in one case sample_id was being used instead of sample_name and it went undiscovered, or similar.

Well_description became 'well_description' in the sample files because at least in some places the section is not case-insensitive. I elected to change it to well_description and make it an issue to ensure column names could be case-insensitive throughout the code.